kotlin-code-style.md

```diff
@@ -40,49 +29,43 @@ class Person(val firstName: String, val lastName: String, var age: Int) {
 }
 ```
 
-# Extensions
+## Constructors
 
-When adding extensions to external classes, make separate files for each type
-and name them _<Type>Extensions.kt_:
+As often as possible we should specify if the constructor properties are mutable
+(`var`) or read-only (`val`):
```

Comment:
I agree, I didn't formulate my thoughts clearly. I meant we should make the primary constructor arguments accessible instead of doing something like this : 

```
class Person(first: String, last: String) {
    val firstName : String
    val lastName : String

    init {
        firstName = first
        lastName = last
    }

}
```